### DISCUSSION TITLE
工研院前進CES 展出運動科技、AI及機器人創新技術

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#565] [科技新知]
這個設計彷彿只有在電影出現過吧，居然在我還活著的時候就現實化了，真的是太神奇了2022
https://event.itri.org/ces2023/tw/tech_dt_i_meta_wear.html

**Comment by awehao** [comment]
若是配合VR實境，感覺又是一個重大突破，撇開電動體驗，若是真正運動時這樣搞，哪叫運動啊，沒料

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Raywei0214** [comment]
工研院的出現，重寫了人生的意義。
博人在不經意間這樣說過，一次失敗，只是證明我們成功的決心還夠堅強。這句話讓我們得到了一個全新的觀點去思考這個問題。

**Comment by manipulateed** [comment]
這個科技太酷了吧，讓從未敢跨出舒適圈的人，在家就可以體驗家附近或是不敢在外獻醜的我們，有更好的機會去嘗試這些運動，還可以減燒真實運動帶來的傷害的可能。
